installer_test.go

```diff
@@ -34,6 +36,39 @@ func TestInstaller_Ensure(t *testing.T) {
 	}
 }
 
+func TestInstaller_Ensure_findable(t *testing.T) {
+	testutil.EndToEndTest(t)
+
+	originalPath := os.Getenv("PATH")
+	os.Setenv("PATH", "")
+	t.Cleanup(func() {
+		os.Setenv("PATH", originalPath)
+	})
+
+	dirPath, fileName := testutil.CreateTempFile(t, "")
+	os.Setenv("PATH", dirPath)
```

Comment:
In case it may be of interest, Go 1.17 includes [`(testing.T).Setenv(key, value string)`](https://pkg.go.dev/testing#T.Setenv) to do this automatically 😄 

```suggestion
	dirPath, fileName := testutil.CreateTempFile(t, "")
	t.Setenv("PATH", dirPath)
```

Reply:
Ha, TIL about `T.Setenv` - thanks!